src/ert/simulator/forward_model_status.py

```diff
@@ -119,7 +119,7 @@ def try_load(cls, path: str) -> "ForwardModelStatus":
         status = cls(status_data["run_id"], start_time, end_time=end_time)
 
         for job, state in zip(job_data["jobList"], status_data["jobs"]):
```

Comment:
job* -> fm_step (but maybe not the string "jobs" here as that might refer to jobs.json?)